docs/src/index.md

```diff
@@ -33,37 +33,77 @@ The following example shows how to train a `KNNClassifier` on the crabs dataset.
 ```julia
 using NearestNeighborModels, MLJ
 X, y = @load_crabs; # loads the crabs dataset from MLJ
-train_inds, test_inds = partition(1:nrows(X), 0.7, shuffle=false, rng=100);
+train_inds, test_inds = partition(1:nrows(X), 0.7, shuffle=false);
```

Comment:
(just FYI: You can leave `shuffle=false` out, as it's the default; no harm leaving it in for clarity)